package.json

```diff
@@ -34,6 +34,7 @@
     "@dfinity/principal": "^0.9.3",
     "@hookform/error-message": "^2.0.0",
     "@psychedelic/plug-controller": "^0.17.4",
+    "@psychedelic/plug-controller-beta": "^0.18.5",
```

Comment:
you sure you wanna merge this ?

Reply:
this pr will be merged to the wallet-connect branch, this is not a final work. But i dont know, what do you think @aleacevedo?